test/standalone/Bucket/albedo_types.jl

```diff
@@ -328,7 +328,7 @@ end
             ref_time,
             h_atmos,
         )
-        Δt = FT(1.0)
+        τc = FT(1.0)
```

Comment:
I renamed these bc at first it was confusing to me why `\Delta t` was being defined with `FT`